### PR TITLE
W-14418702 Clarify certificate renewal docs (1.5)

### PR DESCRIPTION
**Pull request by chris-campbell9**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: gateway/1.5/modules/ROOT/pages/flex-gateway-renew-certificate.adoc [hunk 1]
```diff
@@ -123,14 +131,7 @@ registration renew \
...
 
-[NOTE]
-====
-Unsuccessful renewal attempts generate an empty `registration.yaml` file in the current path. Running `renew` again with this empty file in the current path results in the following error:
-
-`Error: file already exists: /renew/registration.yaml`
-
-Delete the empty `registration.yaml` file.
-====
+NOTE: In cases where the location of the created registration file differs from the runtime registration file location, you must copy the newly created file to the runtime registration file location.
```
Comment (Crispy-Salesforce) [inline gateway/1.5/modules/ROOT/pages/flex-gateway-renew-certificate.adoc]: I'd try to make it simpler:
"NOTE: If the location of the created registration file differs from the runtime registration file location, you must copy the newly created file to the runtime registration file location."

File: gateway/1.5/modules/ROOT/pages/flex-gateway-renew-certificate.adoc [hunk 2]
```diff
@@ -224,14 +233,7 @@ registration renew \
...
```
Comment (Crispy-Salesforce) [inline gateway/1.5/modules/ROOT/pages/flex-gateway-renew-certificate.adoc]: Same comment as above.

**Review by Crispy-Salesforce — DISMISSED** [review]

File: gateway/1.5/modules/ROOT/pages/flex-gateway-renew-certificate.adoc (hunk 1 above)
Comment (chris-campbell9) [reply gateway/1.5/modules/ROOT/pages/flex-gateway-renew-certificate.adoc]: Good catch! Fixed.

File: gateway/1.5/modules/ROOT/pages/flex-gateway-renew-certificate.adoc (hunk 2 above)
Comment (chris-campbell9) [reply gateway/1.5/modules/ROOT/pages/flex-gateway-renew-certificate.adoc]: Fixed!

**Review by Crispy-Salesforce — APPROVED** [review]

**Pull request merged**